pkg/scalers/prometheus_scaler.go

```diff
@@ -134,6 +144,18 @@ func parsePrometheusMetadata(config *ScalerConfig) (meta *prometheusMetadata, er
 		meta.cortexOrgID = val
 	}
 
+	meta.ignoreNullValues = defaultIgnoreNullValues
+	if val, ok := config.TriggerMetadata[ignoreNullValues]; ok && val != "" {
+		ignoreNullValues, err := strconv.ParseBool(val)
+		if err != nil {
+			return nil, fmt.Errorf("err incorrect value for ignoreNullValues given: %s, " +
+				"please using ture or false", val)
+		}
+		meta.ignoreNullValues = ignoreNullValues
+	} else {
+		meta.ignoreNullValues = defaultIgnoreNullValues
+	}
```

Comment:
this else if not needed because the default is set in line 147

Reply:
sorry, my bad